samples/zigbee/ncp/src/main.c

```diff
@@ -9,13 +9,60 @@
  */
 
 #include <drivers/uart.h>
+#include <drivers/gpio.h>
 #include <usb/usb_device.h>
 #include <logging/log.h>
 #include <zb_nrf_platform.h>
 #include <zb_osif_ext.h>
+#include <dfu/mcuboot.h>
+#include <zephyr.h>
+#include <device.h>
 
 LOG_MODULE_REGISTER(app);
 
+#if DT_NODE_EXISTS(DT_ALIAS(rst0))
```

Comment:
Do we want to enable this variant by default?
I think we should either modify the `nrf/samples/zigbee/ncp/child_image/mcuboot_usb.conf` to use the GPIO trigger or provide an overlay to show how to use this feature.

Either way - a small update in the docs is required too :slightly_smiling_face: